strawberry-graphql-django package integration (#949)
strawberry-graphql/strawberry: RELEASE.md

release type: patch

> This release integrates `strawberry-graphql-django` package into strawberry core so that it is possible to use django package directly through `strawberry.django` namespace.

Reviewer: ```suggestion
Integrate the `strawberry-graphql-django` package into Strawberry core so that it's 
possible to use the Django extension package directly through the `strawberry.django`
namespace.
```

release type: patch

> Integrate the `strawberry-graphql-django` package into Strawberry core so that it's
> possible to use the Django extension package directly through the `strawberry.django`
> namespace.

Reviewer 2: ```suggestion
This release tntegrates the `strawberry-graphql-django` package into Strawberry core so that it's possible to use the Django extension package directly via the `strawberry.django`
namespace.

You still need to install `strawberry-graphql-django` if you want to use the extended django support.

See: https://github.com/strawberry-graphql/strawberry-graphql-django
```

Author: RELEASE.md updated
